finance/proposal-calls/cycle3/user-dev-support-rse.md

```diff
@@ -0,0 +1,82 @@
+### Title
+
+Research Software Engineer: User and Developer Support
+
+### Project Team
+
+- Moritz Guenther
+- Adrian Price-Whelan (original plan)
+
+
+### Project Description
+
+A research software engineer (RSE) position to
+tackle both user-facing and developer-facing support issues, as identified and
+prioritized by the Astropy community and infrastructure teams. The RSE would have a
+"manager" who is an existing Astropy contributor with existing knowledge of the Astropy
+core package, ecosystem, infrastructure packages, and community. The RSE is expected to
+eventually work independently, but with feedback from the Manager and Astropy community.
+We envision about an equivalent ~50% FTE appointment, but that can be scaled up or down
+based on the available budget.
+
+### Project / Work
+
+The Astropy Project is seeking to hire a full-time or part-time research software engineer on a fixed-time contract to develop and implement new features that the community has identified as high priorities in the roadmap but that the volunteer contributors do not have the capacity to deliver, as well as address smaller, but long-standing bugs and issues in the astropy core package and the coordinated packages.
+
+The broad scope of work for the RSE includes any efforts to support both Astropy users
+and contributors,
+implement more specific projects related to Astropy core subpackages (some examples
+given below). In particular, the RSE shall tackle a number of long-standing issues and
+bugs as listed in the issue trackers for the core package and the coordinated packages;
+more bugs and issues are available than can be addressed by this position, so the
+choice of which ones to address will be guided by community input, and the experience
+and expertise of the RSE, under the manager's supervision. To reduce the burden on volunteer
+maintainers, the funded manager will perform a first review where appropriate, to keep the
+effort needed for the sub-package maintainers to a minimum.
+
+The RSE position is distinct from the Software Operational Support Specialist (SOSS)
+position in that the RSE is expected to take on and complete discrete *projects*, rather
+than provide continuous support (e.g., monitoring issues, helping new contributors,
+etc., as was included in the SOSS scope of work).
+
+On the other hand, the RSE shall work on the items listed on [Astropy
+Roadmap](https://github.com/astropy/astropy-project/blob/main/roadmap/roadmap.md), that the
+community has already prioritized, but not yet found the resources to implement.
+
+Example projects:
+* *Restructure the Astropy developer documentation*: The current developer documentation
```

Comment:
If these are ordered by priority, I would move the documentation to the bottom. I'd rather this hire focus on benchmark or integration testing that we sorely need.

Reply:
This was not intended to be ordered by priority, but I will re-order.